strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileAttributeType.java

```diff
@@ -0,0 +1,42 @@
+package org.carlspring.strongbox.providers.io;
+
+import java.util.Arrays;
+
+public enum RepositoryFileAttributeType
```

Comment:
Question: Didn't we already have a similar class for the file system? Have you extracted the code here, or is this a different thing (and I'm missing something)?